Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: format
 format:
-	black ./apis/* ./models/* ./test/* ./visualizations/*
+	black ./apis/* ./models/* ./test/* ./visualizations/* --line-length=80
```

Comment:
Changes standard formatting line length to 80 instead of the 88 that it defaults to. This is also the value that the linter uses, so they now work together.